src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -523,7 +523,7 @@ public String getQuestionResultStatisticsCsv(
         DecimalFormat df = new DecimalFormat("#.##");
         
         for(Entry<String, Integer[]> entry : optionTotalCount.entrySet() ){
-            double average = entry.getValue()[0]/entry.getValue()[1];
+            double average = (double)entry.getValue()[0]/entry.getValue()[1];
```

Comment:
Does the webpage and csv both use the same function to calculate the average? if not, they should.